Program.cs

```diff
@@ -1,10 +1,13 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
+﻿//An isogram is a word that has no duplicate letters.
+//Create a function that takes a string and returns either true or false depending on whether it's an "isogram". 
+
+Console.WriteLine ("Enter a string: ");
+string input = Console.ReadLine (); string output = "";
+if (isogram () == true) Console.WriteLine ("It is an isogram.");
+else Console.WriteLine ("Not an isogram.");
+
+bool isogram () {
+   foreach (char item in input) {if (output.Contains (item) == false) output += item;}

```

Comment:
This can be modified if out contains the char item then no need to iterate till the last char of the string. You can break and print the result.
Use If(!....) instead of checking it with false.